packages/flutter_image/example/lib/main.dart

```diff
@@ -0,0 +1,75 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter_image/flutter_image.dart';
+
+void main() => runApp(const MyApp());
+
+/// The main app.
+class MyApp extends StatelessWidget {
+  /// Contructs main app.
+  const MyApp({super.key});
+
+  /// Returns the URL to load an asset from this example app as a network source.
+  String getUrlForAssetAsNetworkSource(String assetKey) {
+    return 'https://github.com/flutter/packages/blob/main/packages/flutter_image/example/$assetKey?raw=true';
```

Comment:
This needs to use a pinned version, not `main`, so that it's not subject to unexpected breakage.

Reply:
By pinned version do you mean this 2e1673307ff7454aff40b47024eaed49a9e77e81?

Reply:
That's an example of a pinned version, yes.